<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Szakdolgozat_AWFRNEClass</class>
 <widget class="QMainWindow" name="Szakdolgozat_AWFRNEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Szakdolgozat_AWFRNE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>770</width>
        <height>770</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonAddPoint">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add new point!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonInterpolation">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Interpolate!</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditX">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string extracomment="X koordinÃ¡ta"/>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditY">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelMessage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelXCoord">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelYCoord">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxShowCoords">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show coordinates over cursor</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInterpolation">
    <property name="title">
     <string>Interpolation</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuDerivatives">
    <property name="title">
     <string>Derivatives</string>
    </property>
   </widget>
   <addaction name="menuInterpolation"/>
   <addaction name="menuDerivatives"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Szakdolgozat_AWFRNE.qrc"/>
 </resources>
 <connections/>
</ui>
